static/app/components/events/eventStatisticalDetector/breakpointChart.tsx

```diff
@@ -29,6 +29,52 @@ type EventBreakpointChartProps = {
   event: Event;
 };
 
+type BreakpointChartData = {
+  event: Event;
+  eventStatsReponse:
+    | {
+        data: EventsStatsData;
+        meta: MetaType;
+      }
+    | undefined;
+};
+function getBreakPointChartPropsFromData(chartData: BreakpointChartData) {
+  const event = chartData.event;
+  const {breakpoint} = event?.occurrence?.evidenceData ?? {};
+  const datetime = getDateTimeFromBreakPoint(breakpoint);
```

Comment:
does `getDateTimeFromBreakPoint` handle empty states well, like `{}` as we get for the fallback?

Reply:
I'll type it better.